presto-main/src/main/java/io/prestosql/execution/QueryStateMachine.java

```diff
@@ -536,9 +535,6 @@ private QueryStats getQueryStats(Optional<StageInfo> rootStage)
 
             PlanFragment plan = stageInfo.getPlan();
             if (plan != null && plan.getPartitionedSourceNodes().stream().anyMatch(TableScanNode.class::isInstance)) {
```

Comment:
We probably should think about removing this if too, but not as part of this PR

Reply:
In fact, we probably should change semantics of `rawInputDataSize`, `rawInputPositions`, `processedInputDataSize` and `processedInputPositions` before we remove `rawInputDataSize` and `rawInputPositions` altogether. This is because there is no equivalence for `rawInputDataSize` once we remove it and some listeners might rely on it (e.g `SplitCompletedEvent`)